WindowBoard.h

```diff
@@ -49,6 +52,7 @@ class GameWindow : public BWindow
 private:
 	NumberView **		fViews;
 	WindowBoard *		fMaster;
+	HighscoreWindow*	fHighscoreWindow;
```

Comment:
this is not needed, you can use a local variable in the function where the window is created

Reply:
Got it, now using a local variable instead.

Reply:
Won't using a local variable cause a discrepancy?
The object will get created, and it'll go out of scope instantly. Which will destroy the window?

Reply:
No, the object is created with "new BWindow" and remains allocated. It doesn't matter that you app has no pointers to it. The window (and all its contained views) is destroyed when it is closed by the user.

Reply:
Okay! This probably was the reason for the crash, because I wasn't allocating it with pointer. Still some error remains which is causing the crash.